o8g/Extractor/config.xml

```diff
@@ -0,0 +1,76 @@
+﻿<xml>
+  <game cardsUrl="https://api.fiveringsdb.com/cards"
+        packsUrl="https://api.fiveringsdb.com/packs"
+        gameGuid="bb0f02e7-2a6f-4ae3-84a2-c501b4176844"
+        packId="id"
+        cardNumber="position"
+        cardOctgnNumber="Card Number"
+        cardName="name"
+        cardPackId="pack_code"
+        cardId="octgn_id"
+        cardImage="image_url"
+        />
+  <images>
+    <image name="FiveringsDB" url="{0}" />
+  </images>
+  <properties>
+    <property octgn_name="Clan" type="property" value="clan" capitalize="True" />
+    <property octgn_name="Text" type="property" value="text" isRich="True"/>
+    <property octgn_name="Type" type="property" value="type" capitalize="True" />
+    <property octgn_name="Traits" type="property" value="traits" capitalize="True" delimiter=". " />
+    <property octgn_name="Ring" type="property" value="element" capitalize="True" />
+    <property octgn_name="Card Number" type="property" value="position" />
+    <property octgn_name="Cost" type="property" value="cost" />
+    <property octgn_name="Military Skill" type="property" value="military" />
+    <property octgn_name="Military Skill" type="property" value="military_bonus" />
```

Comment:
octgn name should be Bonus Military Skill. I don't mind adjusting these things but I'm not sure how to update the generated xml yet

Reply:
Thoughts on combining "Military Skill" and "Bonus Military Skill" to one field (same with political/strength)?  The + in front of the value seems to consistently point to the Bonus, so I feel like its pretty obvious telling them apart already.  Plus it saves some space in the deck editor column-wise